avocado/core/dispatcher.py

```diff
@@ -71,8 +72,17 @@ def settings_section(self):
         return "plugins.%s" % self.plugin_type()
 
     def enabled(self, extension):
-        disabled = settings.get_value('plugins', 'disable', key_type=list)
-        return self.fully_qualified_name(extension) not in disabled
+        """
+        Checks configuration for explicit mention of plugin in a disable list
+
+        If configuration section or key doesn't exist, it means no plugin
+        is disabled.
+        """
+        try:
+            disabled = settings.get_value('plugins', 'disable', key_type=list)
+            return self.fully_qualified_name(extension) not in disabled
+        except SettingsError:
+            return False
```

Comment:
Oh my, obviously it it! That's why no plugins are loaded (and I thought it was due to a different issue, which I was going to investigate later). Thanks for catching this!